src/App.css

```diff
@@ -1,22 +1,73 @@
-.App {
+
+.app {
   text-align: center;
+  display: flex;
+  height: 90%;
 }
 
-.App-logo {
-  height: 40vmin;
+.app-header {
+  height: 10%;
+  background-color: #1a1b1b;
+  display: flex;
+  margin-bottom: 5px;
 }
 
-.App-header {
-  background-color: #282c34;
-  min-height: 100vh;
+.heading {
+  font-size: xx-large;
```

Comment:
@MehwishDogar using a shorthand property like `font:` is considered a good practice rather then individual properties like `font-family`.
https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties

Reply:
I still can't see all the font properties merged into one single shorthand property `font`.